{"ts":"2019-03-03 20:42:32.936+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#createTodo","thread":"qtp800491893-30"},"msg":"SQL Error: 0, SQLState: 42501"}
{"ts":"2019-03-03 20:42:32.940+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#createTodo","thread":"qtp800491893-30"},"msg":"ERROR: permission denied for table todo"}
{"ts":"2019-03-03 20:42:32.950+0300","lvl":"ERROR","logger":"org.hibernate.internal.ExceptionMapperStandardImpl","mdc":{"rid":"NoRequestId","controller":"TodoResource#createTodo","thread":"qtp800491893-30"},"msg":"HHH000346: Error during managed flush [org.hibernate.exception.SQLGrammarException: could not execute statement]"}
{"ts":"2019-03-03 20:42:47.185+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-33"},"msg":"SQL Error: 0, SQLState: 42501"}
{"ts":"2019-03-03 20:42:47.186+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-33"},"msg":"ERROR: permission denied for table todo"}
{"ts":"2019-03-03 20:47:59.947+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-35"},"msg":"SQL Error: 0, SQLState: 42501"}
{"ts":"2019-03-03 20:47:59.947+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-35"},"msg":"ERROR: permission denied for table todo"}
{"ts":"2019-03-03 20:50:21.737+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-17"},"msg":"SQL Error: 0, SQLState: 42501"}
{"ts":"2019-03-03 20:50:21.738+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp800491893-17"},"msg":"ERROR: permission denied for table todo"}
{"ts":"2019-03-05 01:32:13.357+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp1662035774-20"},"msg":"SQL Error: 0, SQLState: 42703"}
{"ts":"2019-03-05 01:32:13.361+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#getAllTodos","thread":"qtp1662035774-20"},"msg":"ERROR: column todo0_.createdat does not exist\n  Hint: Perhaps you meant to reference the column \"todo0_.created_at\".\n  Position: 63"}
{"ts":"2019-03-05 01:32:26.765+0300","lvl":"WARN","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#createTodo","thread":"qtp1662035774-33"},"msg":"SQL Error: 0, SQLState: 42703"}
{"ts":"2019-03-05 01:32:26.766+0300","lvl":"ERROR","logger":"org.hibernate.engine.jdbc.spi.SqlExceptionHelper","mdc":{"rid":"NoRequestId","controller":"TodoResource#createTodo","thread":"qtp1662035774-33"},"msg":"ERROR: column \"createdat\" of relation \"todo\" does not exist\n  Position: 30"}
{"ts":"2019-03-05 02:00:59.342+0300","lvl":"ERROR","logger":"org.hibernate.internal.ExceptionMapperStandardImpl","mdc":{"rid":"NoRequestId","controller":"TodoResource#updateTodo","thread":"qtp700996919-34"},"msg":"HHH000346: Error during managed flush [org.hibernate.HibernateException: identifier of an instance of ru.hh.school.checkup.entities.Todo was altered from 1 to null]"}
